eclcli/bare/v2/server.py

```diff
@@ -548,13 +548,26 @@ def get_parser(self, prog_name):
             metavar="<server>",
             help="Name or ID of server",
         )
+        parser.add_argument(
+            "--type",
+            metavar="<type>",
+            help="Server shutdown mode. A valid value is HARD (IPMI) or SOFT (ACPI)",
+        )
```

Comment:
You can add a validator easier like this.
```suggestion
            choices=["SOFT", "HARD"],
        )
```